OSOE-548: Upgrade to Orchard Core 1.6 in Orchard-Azure-Application-Insights

Psichorex opened the pull request:
[OSOE-548](https://lombiq.atlassian.net/browse/OSOE-548)

[OSOE-548]: https://lombiq.atlassian.net/browse/OSOE-548?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Psichorex committed:
Updating to OC 1.6

Psichorex committed:
Upgrading Lombiq.ChartJs.Tests.UI package.

Piedone commented on Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI/Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI.csproj:
```diff
@@ -28,7 +28,7 @@
   <ItemGroup Condition="'$(NuGetBuild)' == 'true'">
-    <PackageReference Include="Lombiq.Tests.UI" Version="6.0.1" />
+    <PackageReference Include="Lombiq.Tests.UI" Version="6.0.2-alpha.1.osoe-548" />
     <PackageReference Include="Lombiq.Privacy.Tests.UI" Version="5.0.2" />
```
Lombiq.Privacy.Tests.UI needs to be updated too.

Psichorex committed:
Updating to latest Lombiq alpha releases.

Psichorex committed:
Removing mistakenly included "v" from versions.

Psichorex committed:
Updating to latest Lombiq.Tests.UI alpha release.

dministro committed:
Upgrading Lombiq.HelpfulLibraries.* package reference version
Upgrading Lombiq.Tests.UI.* package reference version
Upgrading Lombiq.Privacy.* package reference version

dministro committed:
Merge branch 'dev' of https://github.com/Lombiq/Orchard-Azure-Application-Insights into issue/OSOE-548

dministro committed:
Merge branch 'issue/OSOE-548' of https://github.com/Lombiq/Orchard-Azure-Application-Insights into issue/OSOE-641

Piedone committed:
Merge pull request #60 from Lombiq/issue/OSOE-641

OSOE-641: Fix remaining issues for the Orchard Core 1.6 upgrade in Orchard-Azure-Application-Insights

Piedone reviewed (approved):
Just marking the merge from `issue/OSOE-641`.

Piedone committed:
Updating Lombiq.HelpfulLibraries NuGet reference to latest alpha

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-548

# Conflicts:
#	Lombiq.Hosting.Azure.ApplicationInsights/Lombiq.Hosting.Azure.ApplicationInsights.csproj

Piedone committed:
Merge remote-tracking branch 'origin/dev' into issue/OSOE-548

# Conflicts:
#	Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI/Lombiq.Hosting.Azure.ApplicationInsights.Tests.UI.csproj

Piedone committed:
Updating Lombiq NuGet references

Piedone committed:
Updating Lombiq NuGet references to release versions
